apps/api/src/app/workflows-v2/mappers/notification-template-mapper.ts

```diff
@@ -15,7 +15,10 @@ import { NotificationStepEntity, NotificationTemplateEntity } from '@novu/dal';
 import { WorkflowInternalResponseDto } from '@novu/application-generic';
 import { buildSlug } from '../../shared/helpers/build-slug';
 
-export function toResponseWorkflowDto(workflow: WorkflowInternalResponseDto): WorkflowResponseDto {
+export function toResponseWorkflowDto(
+  workflow: WorkflowInternalResponseDto,
+  fullSteps: StepDataDto[]
```

Comment:
personal opinion we can keep `steps` name, and the type should explain what we get here.

```suggestion
  steps: StepDataDto[]
```